.github/CIP-TEMPLATE.md

```diff
@@ -7,7 +7,7 @@ Authors:
     - John Doe <john.doe@email.domain>
 Implementors: []
 Discussions:
-    - https://github.com/cardano-foundation/cips/pulls/?
+    - https://github.com/cardano-foundation/CIPs/pull/?
```

Comment:
```suggestion
  - https://github.com/cardano-foundation/CIPs/pull/?
```
should these be 2 or 4 spaces? (same for author too)

Reply:
@Ryun1 technically it doesn't matter.  Generally we see 2, 4, or 8 spaces... the only things that matter to list formatting are when the number of spaces increase (which indents) and when they line up with a previous number of spaces (which outdents).

Personally I've always stuck with the standard first posted by the document author, unless indenting an outline structure for the first time: in which case I use a 2-space increment.

A bunch of @KtorZ authored CIPs have the 4-space standard which is fine with me as well.  I don't think we'd be saving ourselves any grief if we tried to mandate or standardise on a particular spacing increment. 🤔

Reply:
> technically it doesn't matter

ahh good, I hadnt found any examples of it breaking things, but was unsure.

>  don't think we'd be saving ourselves any grief

Agreed, it was just something that has bugged me and I wanted to raise. 
although it is of course very insignificant